### ISSUE TITLE
Change some fields from required to optional

### ISSUE THREAD

**Issue by fureigh** [18F/forest-service-prototype#20]
- Address 2
- Day/evening phone (require one but not both)

...etc.

**Comment by fureigh** [comment]
@lauraGgit This appears to have been covered in #17 — accurate?

**Comment by lauraGgit** [comment]
Yes, but I haven't been able to get the spectator number to be zero - even
if its in the `models.py`
